contracts/CappedMintableToken.sol

```diff
@@ -0,0 +1,12 @@
+pragma solidity ^0.5.0;
+
+import "../node_modules/openzeppelin-solidity/contracts/token/ERC20/ERC20Capped.sol";
+
+contract CappedMintableToken is ERC20Capped{
+
+    constructor(uint256 cap) ERC20Capped(cap) public {}
+
+    function mint(address _to, uint256 _amount) public returns (bool){
```

Comment:
you can remove this function because it will be inherited from the `ERC20Capped `